ChangeLog

```diff
@@ -130,6 +130,8 @@ modules are added.
 * New checker ``invalid-all-format``. Emitted when ``__all__`` has an invalid format,
   i.e. isn't a ``tuple`` or ``list``.
 
+* Add test cases for nested exceptions in ``unused-variable`` error message.
+
```

Comment:
```suggestion
```
It's an internal maintenance MR you're starting to do the silent unseen work :wink: